CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
 - Fix: allow use JEXL expresions in explicitAttrs for conditional propagation of measures (reopen #1179, for Devices)
+- Fix: replace findType by findTypeSilently to avoid raise mongo alarm (DEVICE_GROUP_NOT_FOUND) before launch op against CB 
```

Comment:
Maybe more "user friendly" this way:

```suggestion
- Fix: avoid raise mongo alarm (DEVICE_GROUP_NOT_FOUND) before launch op against CB 
```

The user doesn't know nothing about internal functions such as findType or findTypeSilently...

Reply:
Btw, no issue associated?

Reply:
No issue related was opened